app/assets/stylesheets/custom.css.scss

```diff
@@ -5,6 +5,13 @@
 
 $gray-medium-light: #eaeaea;
 $white: #ffffff;
+$red: #ff0000;
+
+@mixin box_sizing {
```

Comment:
Name of mixin `box_sizing` should be written in all lowercase letters with hyphens instead of underscores